tests/kabir_practice/matrix_multiplier.futil.data

```diff
@@ -0,0 +1,42 @@
+{
+    "m1": {
+        "data": [
+            [1, 2, 3],
+            [3, 4, 5],
+            [5, 6, 7]
```

Comment:
For readability and easy comparison with the .expect file, consider formatting this file like the .expect file. I have an autoformatter set up, so on my computer this is just a matter of telling VSCode that the file is a JSON file (not plaintext) and then saving.